sync_image_ghcr.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -e
+# Simple script that pushes missing/outdated tags from an image in DockerHub to GHCR, skipping up-to-date tags 
+# GitHub package registry. You need to be logged in using 'docker login' first: https://docs.github.com/es/packages/working-with-a-github-packages-registry/working-with-the-container-registry
+target_repo="ghcr.io"
+tag_file="tag_list.txt"
+original_image="$1"
+rm -rf $tag_file
+url="https://hub.docker.com/v2/repositories/${original_image}/tags"
+tag_count=$(wget -q "$url" -O - | jq -r '.count')
+
+echo "Fetching tags from DockerHub..."
+while true; do
+	results=$(wget -q "$url" -O - | jq -r '.')
```

Comment:
so this will loop at most ~100 times due to API rate limits (Thank you Docker Inc. :roll_eyes:), which would lead to the script abruptly exiting due to you having a if check in L30

(at least I am fairly certain the `tags`endpoint counts toward that quota too)

Reply:
Nope, this one doesn't have any quota.

I tested this locally and it queried over 300 pages without any issues. The problem is down below, when ``docker pull`` runs 😁.

As I mentioned in the OP, this workflow will fail only on the pull or push process due to API constraints, but that's not problematic at all as the work it's going to be resumed in the next scheduled run.

I also expect a lot of API problems in the first weeks of this workflow running, but once GHCR gets up-to-date, it won't be a problem, as we only need to sync the images that have been published in the last 24 hours.

Reply:
Okay, interesting. I could have sworn that any GET against the API was counting towards that cap, but this is good news if it does not :tada: 

Yeah as soon as the Pull starts this will be 'FUN'! 
However, on a somewhat related note how long do we keep / or intent to keep unstable builds available?

Reply:
With unstable you mean this method of syncing? Or the unstable images that we're pushing daily. AFAIK we're storing it locally in our infraestructre VPS for 2 days, while in DockerHub (and now GHCR once this is merged) it's since the beginning of Jellyfin! 😋

Reply:
I was referring to to the actual images that get build not only daily but  on each push to master (web or server).

But given your answer for that ... we should consider some form of cleanup strategy for those IMHO. (Irrelevant of the registry)